packages/@pollyjs/core/src/utils/normalize-request.js

```diff
@@ -8,15 +9,23 @@ const { keys } = Object;
 const { isArray } = Array;
 const { parse } = JSON;
 
-export function method(method) {
-  return (method || 'GET').toUpperCase();
+export function method(method, config) {
```

Comment:
is method not always provided which is why we have the fallback?  Should we fix that upstream?

Reply:
It is 😄. Updated.